tests/end_to_end.rs

```diff
@@ -214,15 +215,19 @@ async fn test_create_table() {
     let results = list_columns_query(&context).await;
```

Comment:
Perhaps better to omit the system schema from this and `list_tables_query`, and rely on explicit tests instead?

Reply:
Maybe, but other tests that don't care about `information_schema` (e.g. `test_create_external_table_http`) have it in there as well, so let's leave it like that.